internal/datastore/crdb/tx.go

```diff
@@ -61,10 +64,17 @@ func executeWithResets(ctx context.Context, fn innerFunc, maxRetries uint8) (err
 	for retries = 0; retries <= maxRetries; retries++ {
 		err = fn(ctx)
 		if resettable(ctx, err) {
-			log.Ctx(ctx).Warn().Err(err).Msg("retrying resetteable database error")
+			after := retry.BackoffExponentialWithJitter(100*time.Millisecond, 0.5)(uint(retries))
```

Comment:
Once this lands, I'd suggest opening an issue to make this parameterizable.

Reply:
+1